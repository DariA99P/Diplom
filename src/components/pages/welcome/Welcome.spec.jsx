import React from 'react';
import renderer from 'react-test-renderer';

import Welcome from '.';

describe('<Welcome />', () => {
  let tree;
  beforeEach(() => {
    global.fetch = jest.fn(() => Promise.resolve({ text: () => '# Hello' }));
    tree = renderer.create(<Welcome />).toJSON();
  });

  it('should renders properly', () => {
    expect(tree).toMatchSnapshot();
  });
});
