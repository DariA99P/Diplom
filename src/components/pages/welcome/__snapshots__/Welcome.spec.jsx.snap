// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome /> should renders properly 1`] = `
<div>
  <h1>
    Welcome to UI Boilerplate 2
  </h1>
  <p>
    Please read the README.MD in the folder directory for more details.
  </p>
</div>
`;
